src/services/pocket-relayer.ts

```diff
@@ -440,6 +441,24 @@ export class PocketRelayer {
       let nodes: Node[] = pocketSession.sessionNodes
       const relayStart = process.hrtime()
 
+      const cachedRemovedSessionNodes = await this.redis.get(`session-${pocketSession.sessionKey}`)
+
+      if (cachedRemovedSessionNodes) {
+        const nodesToRemove: string[] = JSON.parse(cachedRemovedSessionNodes)
+
+        nodes = nodes.filter((n) => !nodesToRemove.includes(n.publicKey))
+      } else {
+        // Maximum time in milliseconds for the next session to be rolloved,
```

Comment:
* to be rolled over

I was a bit concerned about using the session length for anything given that block timing is variable. Optimal block timing is 15 minutes but sometimes they take up to 18 minutes.

I'm not sure we need to set this TTL with any sort of specificity. What's the harm if we set it for 2 hours?  It won't be pulled up if the sessionKey changes.